From 431d1dd2a6e5abf9d52fe64b251e913a4f0b875c Mon Sep 17 00:00:00 2001
From: Alex Leibovich <alexl@marvell.com>
Date: Sun, 7 Apr 2019 10:28:57 +0300
Subject: [PATCH 1088/1200] defconfig: cn913x: changed efuse default
 configuration

Added efuse  default configuration.
By default the FW will support "fuse" command,
the efuse will be enabled, and read_only.

Change-Id: Ic107ec75899745644cdb6fd1395796c77837318e
Signed-off-by: Alex Leibovich <alexl@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/7044
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Reviewed-by: Kostya Porotchkin <Kostya.Porotchkin@cavium.com>
---
 configs/mvebu_db_cn91xx_defconfig | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/configs/mvebu_db_cn91xx_defconfig b/configs/mvebu_db_cn91xx_defconfig
index 75af420728..990d790a42 100644
--- a/configs/mvebu_db_cn91xx_defconfig
+++ b/configs/mvebu_db_cn91xx_defconfig
@@ -19,6 +19,7 @@ CONFIG_BOARD_EARLY_INIT_F=y
 CONFIG_SYS_PROMPT="Marvell>> "
 CONFIG_CMD_MVEBU_SAR=y
 # CONFIG_CMD_FLASH is not set
+CONFIG_CMD_FUSE=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
@@ -40,6 +41,8 @@ CONFIG_ENV_IS_IN_SPI_FLASH=y
 CONFIG_SCSI_AHCI=y
 CONFIG_AHCI_MVEBU=y
 CONFIG_BLOCK_CACHE=y
+CONFIG_MVEBU_EFUSE=y
+CONFIG_MVEBU_EFUSE_READ_ONLY=y
 CONFIG_DM_GPIO=y
 CONFIG_DM_PCA953X=y
 CONFIG_DM_I2C=y
@@ -65,9 +68,9 @@ CONFIG_MVEBU_COMPHY_SUPPORT=y
 CONFIG_PINCTRL=y
 CONFIG_PINCTRL_ARMADA_8K=y
 CONFIG_DM_REGULATOR_FIXED=y
+CONFIG_DM_REGULATOR_GPIO=y
 CONFIG_DM_RTC=y
 CONFIG_MARVELL_RTC=y
-CONFIG_DM_REGULATOR_GPIO=y
 # CONFIG_SPL_SERIAL_PRESENT is not set
 CONFIG_DEBUG_UART_BASE=0xf0512000
 CONFIG_DEBUG_UART_CLOCK=200000000
-- 
2.22.0

